safe/engine/test/test_engine.py

```diff
@@ -165,6 +165,7 @@ def test_polygon_hazard_raster_exposure_clipped_grids(self):
                    'and full grids, Got %s and %s' % (x, y))
             assert numpy.allclose(x, y)
 
+    @unittest.skip('Slow test')
```

Comment:
Instead of always skipping, can we use a condition ? `unittest.skipIf(condition, reason)`
For the condition, we can use a environment variable or a QSettings (depending if are on travis or in QGIS UI)